.github/workflows/main.yml

```diff
@@ -5,14 +5,16 @@ on:
     paths-ignore:
       - '**.md'
     branches:
-      - master
+      - '**'
+      - '*'
     tags:
       - '*'
   pull_request:
     paths-ignore:
       - '**.md'
     branches:
-      - master
+      - '**'
+      - '*'
```

Comment:
Please explain these changes.

Reply:
to make CI work on my branch. This is temporary and will be removed before merging.